servo_org_list:
    - 25
    - 5
    - 0
    - 0